features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NRF5_SDK13/source/nRF5xSecurityManager.h

```diff
@@ -114,7 +112,26 @@ class nRF5xSecurityManager : public SecurityManager
 
         return BLE_ERROR_NONE;
     }
-#endif
+#else // -> NRF_SD_BLE_API_VERSION >= 3
+    /**
+     * @brief  Returns a list of addresses from peers in the stacks bond table.
+     *
+     * @param[in/out]   addresses
+     *                  (on input) @ref Gap::Whitelist_t structure where at
+     *                  most addresses.capacity addresses from bonded peers will
+     *                  be stored.
+     *                  (on output) A copy of the addresses from bonded peers.
+     *
+     * @return
+     *           BLE_ERROR_NONE if successful.
+     */
```

Comment:
Again, it would be good to mention something about what is returned when an error occurs.